src/Symfony/Component/Yaml/Inline.php

```diff
@@ -562,6 +562,8 @@ private static function evaluateScalar($scalar, $flags, $references = array())
                         }
 
                         return;
+                    case 0 === strpos($scalar, '!php/const:'):
+                        return @constant(substr($scalar, 11));
```

Comment:
Understood! So I need to:

1. add a flag like `PARSE_PHP_CONSTANTS` in `Yaml` class

2. either throw a `ParseException` or return `null` when the constant does not exists
   maybe using `PARSE_EXCEPTION_ON_INVALID_TYPE`

3. add the flag `PARSE_PHP_CONSTANTS` to `YamlFileLoader` config.

Is that correct? What about point 2?